libs/market-info/src/components/market-expires/market-expires.tsx

```diff
@@ -1,7 +1,7 @@
-import type { SingleMarketFieldsFragment } from '@vegaprotocol/market-list';
-import { getDateFormat, t } from '@vegaprotocol/react-helpers';
+import { getDateTimeFormat, t } from '@vegaprotocol/react-helpers';
 import { format, isValid, parseISO } from 'date-fns';
 
+import type { SingleMarketFieldsFragment } from '@vegaprotocol/market-list';
 export const EXPIRE_DATE_FORMAT = 'MMM dd';
```

Comment:
question (blocking): do we not need to update this format string? Looks like there is a mix of using date-fns for formatting as well the Intl object. Can we tidy this up to use the later?

Reply:
I kept this format because it was used in console-lite. I don't know the reqs/designs on that one and I didn't want this PR to affect anything else except the console.